pytorch_lightning/plugins/apex.py

```diff
@@ -131,3 +131,11 @@ def clip_gradients(self, grad_clip_val: Union[int, float], optimizer: Optimizer,
     @property
     def norm_clipping_epsilon(self):
         return 1e-5
+
+    def step(self, trainer, optimizer, closure):
```

Comment:
can we make this less ambiguous?
is this for training_step or optimizer_step?

Reply:
done. `optimizer_step `